src/share/doc/javatest/regtest/faq.md

```diff
@@ -1022,6 +1022,59 @@ cannot be deleted.
 jtreg will try hard to delete files in the scratch directory, and will wait
 awhile in case the files go away in a timely manner.
 
+### My tests take a long time to run: how do I find where the time goes?
```

Comment:
You might want to consider breaking up your answer into two parts.  The first part answers the question "My **test** takes a long time to run: how to I find where the time goes?" The second part answers "How do I find the longest-running tests in my test suite?"

Reply:
Noted; thanks

Reply:
Looks good!